etc/ctags.readme.txt

```diff
@@ -1,9 +1,25 @@
-Copy the file ctags_for_gap to ~/.ctags and then run
+Run
+  ctags --version
+to see whether you are running Exuberant Ctags or Universal Ctags.
+Append the content of the file ctags_for_gap to the file
+  ~/.ctags
```

Comment:
Note that none of this is necessary if one just wants to use ctags in GAP, at least with excuberant ctags: we already have a .ctags file (well, symlink) in the gap root dir!

Reply:
Which of course does not work with Universal Ctags *sigh*...
Anyways, I'll add a comment at the top of the file.